src/definitions_cxx.hpp

```diff
@@ -863,6 +879,7 @@ enum CCNumber {
 };
 constexpr int32_t kNumCCNumbersIncludingFake = 124;
 constexpr int32_t kNumRealCCNumbers = 120;
+constexpr int32_t kMaxCCNumber = 127;
```

Comment:
This should be 124 - all CCs above 120 are dedicated commands (like all notes off) and not controllers

Reply:
I've since changed this to a more general midi value range constant kMaxMIDIValue = 127;

If I used a smaller number for Midi CC's as you suggest, is there a possibility someone could try to send a CC above 124?

I use this constant when defining a couple array's to track some info by CC #.

	int32_t previousKnobPos[kMaxMIDIValue + 1];
	uint32_t timeLastCCSent[kMaxMIDIValue + 1];

Reply:
Generally deluge should never send a CC above 120 except as part of a separate command that doesn't need tracking, I said 124 as there are dedicated "CCs" internally for MPE params 

Leaving it as 127 in array definitions is fine